src/ProjectTemplates/Web.ItemTemplates/content/ApiController/.template.config/template.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Web", "ASP.NET" ],
+  "name": "API Controller",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An API Controller with or without read/write actions",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.ApiController",
+  "precedence": "9799",
+  "identity": "Microsoft.AspNetCore.Mvc.ApiController.8.0",
+  "shortName": "apiController",
+  "sourceName": "ValueController",
+  "primaryOutputs": [
+    {
+      "path": "ValueController.cs"
+    }
+  ],
+  "defaultName": "ValueController",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "MyApp.Namespace",
+      "type": "parameter"
+    },
+    "actions": {
+      "description": "create controller with read/write actions",
+      "displayName": "add read/write actions",
```

Comment:
displayName is only used by Visual Studio, and would display exactly this way, casing and all. I think the displayName should be sentence or pascal cased (See Razor Page template)

However, in reality the item dialog as implemented does not support showing parameters anyway.

Reply:
fixed